modules/deployment/src/test/java/org/mule/runtime/module/deployment/logging/LoggingSupportabilityContextTestCase.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.deployment.logging;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static uk.org.lidalia.slf4jtest.LoggingEvent.error;
+
+import org.mule.runtime.core.internal.processor.LoggerMessageProcessor;
+import org.mule.runtime.module.deployment.impl.internal.builder.ApplicationFileBuilder;
+import org.mule.runtime.module.deployment.internal.AbstractApplicationDeploymentTestCase;
+
+import java.util.HashMap;
+import java.util.List;
+
+import io.qameta.allure.Description;
+import static org.hamcrest.Matchers.is;
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
+
+public class LoggingSupportabilityContextTestCase extends AbstractApplicationDeploymentTestCase {
+
+  TestLogger logger = TestLoggerFactory.getTestLogger(LoggerMessageProcessor.class);
+
+  @Parameterized.Parameters(name = "Parallel: {0}")
+  public static List<Boolean> params() {
+    return asList(false);
+  }
```

Comment:
It wouldn't work if you removed this and made the constructor receive no arguments, and just passed false to the constructor of the super class?

Reply:
Just checked and no, it wouldn't work.